### DISCUSSION TITLE
How to disable (x hidden items) message?

### DISCUSSION THREAD

**Discussion by karlivory** [nvim-neo-tree/neo-tree.nvim#401] [Q&A] [answered]
Is there some way to not show the _(x hidden items)_ message:

![alt text](https://i.imgur.com/7mUan9k.png)

I tried settings `renderers.message = {}` in the config:

https://github.com/nvim-neo-tree/neo-tree.nvim/blob/8613c458cd04ddd14135060179bc2bca01e2d2f9/lua/neo-tree/defaults.lua#L182-L185

but that just leaves me with empty lines in place of those messages.

**Comment by cseickel** [comment, marked as answer]
This should be controlled with the `filesystem.filtered_items.show_hidden_count` option:

```lua
    require("neo-tree").setup({
      filesystem = {
        filtered_items = {
          visible = false, -- when true, they will just be displayed differently than normal items
          force_visible_in_empty_folder = true, -- when true, hidden files will be shown if the root folder is otherwise empty
          show_hidden_count = false, -- when true, the number of hidden items in each folder will be shown as the last entry
        }
      }
    })

**Reply by karlivory** [reply]
Thanks!
